src/applications/burials-ez/migrations.js

```diff
@@ -0,0 +1,15 @@
+export default [
+  // 0 -> 1, move user back to address page if country code is bad
```

Comment:
Perhaps link to the GitHub issue for some historical reference?

Reply:
Updated the comment